src/errors.rs

```diff
@@ -29,6 +29,12 @@ pub enum InternalError {
     NoSquareRoots,
     #[error("Elements are not coprime")]
     NotCoprime,
+    #[error("Elements of the multiplicative group ZK* cannot be larger than the RSA modulus")]
+    LargerThanModulus,
+    #[error("Elements of the multiplicative group ZK* cannot be negative")]
+    NegativeElement,
+    #[error("Elements of the multiplicative group  ZK* cannot be zero")]
+    IsZero,
```

Comment:
suggestion: On all of these, is `ZK*` the right shorthand? I've been doing `Z*_N` in internal docs.